modules/framework/galasa-parent/dev.galasa.framework.api.ras/src/main/java/dev/galasa/framework/api/ras/internal/common/RasDetailsQueryParams.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package dev.galasa.framework.api.ras.internal.common;
+
+public class RasDetailsQueryParams {
+    
+    private String detail;
+
+    public RasDetailsQueryParams() {
+        
+    }
+
+    public RasDetailsQueryParams(String detail) {
+        this.detail = detail;
+    }
+
+    public String getDetail() {
+        return detail;
+    }
```

Comment:
Instead of getDetail, maybe we could have a method like `isMethodDetailsRequested()` that returns true if "methods" has been passed into the detail param? That way, we can ask this class if we need to include method details or not

(feel free to think of a better method name!)